ui/package.json

```diff
@@ -8,6 +8,7 @@
     "@material-ui/lab": "^4.0.0-alpha.58",
     "@material-ui/styles": "^4.11.4",
     "blueimp-md5": "^2.18.0",
+    "chokidar": "^3.5.2",
```

Comment:
Why add `chokidar` dependency?

Reply:
![image](https://user-images.githubusercontent.com/6691735/126579977-276a5924-945c-4e59-9174-dc6eaf2d8496.png)
`output from master`

![image](https://user-images.githubusercontent.com/6691735/126580155-de08e1d9-c5dd-4b73-b53a-23d2075d00a3.png)
`output w/ chokidar added`

it doesn't seem to be an issue, just a warning it gave so i decided to add the dependency. let me know if you still want it taken out.

Reply:
I never saw this error, and actually `chokidar` is a transitive dependency (listed in `package-lock.json`). Are you using `yarn` instead of `npm`?

Reply:
No, I'm using npm. I don't have yarn on my system. But it's fine, I probably messed something else up so I can just take it out!

Reply:
Try running `npm ci` in the `ui` folder. There were recent changes to the npm dependencies and you may have old dependencies.

Reply:
Did that, but I end up getting the same error.